src/clib/lib/enkf/enkf_state.cpp

```diff
@@ -302,6 +300,8 @@ static std::pair<fw_load_status, std::string> enkf_state_internalize_results(
 
     enkf_fs_type *sim_fs = run_arg_get_sim_fs(run_arg);
     int last_report = time_map_get_last_step(enkf_fs_get_time_map(sim_fs));
+    // TODO investigate usage of last history: is last_report < 0 possible? is
```

Comment:
I think it's better to create an issue than adding TODO's to code as they are quite hidden.

Reply:
ARGH thank you for catching that! that's a personal note that should never have been merged :grimacing:

Reply:
gonna make a PR that removes this immediately.

Reply:
fixed in https://github.com/equinor/ert/pull/4037